paddle/fluid/platform/dynload/lapack.h

```diff
@@ -23,6 +23,9 @@ limitations under the License. */
 // Note(zhouwei): because lapack doesn't provide appropriate header file.
 // should expose API statement yourself.
 
+#define LAPACK_ROW_MAJOR 101
+#define LAPACK_COL_MAJOR 102
```

Comment:
these 2 macros are not used in any other files

Reply:
done